SW/main/burner_on.py

```diff
@@ -0,0 +1,78 @@
+import Stove
+import numpy as np
+
+class BurnerOn:
+    def __init__(self):
+        self.i = 0
+        self.decreasing = []
+        self.off = True
+        
+    def decreasing_temp(self, average):
+        while self.i < 360:
+            if (self.i % 30 == 0):
+                self.decreasing.append(average)
+            self.i += 1
+            break
+
+        if len(self.decreasing) == 12:
```

Comment:
lots of magic numbers here -- 360, 12 30, etc. 

Make them named variables so I have a chance of following along.